sdk/src/main/kotlin/io/rover/sdk/Rover.kt

```diff
@@ -180,10 +183,27 @@ open class Rover(
             HttpClient.installSaneGlobalHttpCache(applicationContext)
         }
 
+        @JvmStatic
+        @JvmOverloads
         fun initialize(application: Application, accountToken: String, @ColorInt chromeTabColor: Int = Color.BLACK) {
             shared = Rover(application = application, accountToken = accountToken, chromeTabBackgroundColor = chromeTabColor)
         }
 
+        /**
+         * Sets an experience transformer on the [Rover] object enabling retrieved experiences to be altered
+         * in the desired way. The transformer is called on the UI thread so the transformer shouldn't block
+         * the thread.
+         */
+        @JvmStatic
+        fun setExperienceTransformer(experienceTransformer: (Experience) -> Experience) {
+            shared?.let { it.experienceTransformer = experienceTransformer } ?: log.w("Rover should be initialised before setting experience transformer.")
+        }
+
+        @JvmStatic
+        fun removeExperienceTransformer() {
+            shared?.experienceTransformer = null
+        }
+
```

Comment:
consider moving this into .shared instead of wrapping it with static-scope accessor method.